src/lib/components/ExpandNavigation.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+	export let isCollapsed: boolean;
+	export let classNames: string;
+</script>
+
+<button
+	on:click
+	class="{classNames} group absolute flex h-16 w-6 flex-col items-center justify-center -space-y-1 outline-none *:h-3 *:w-1 *:rounded-full *:hover:bg-gray-300 max-md:hidden dark:*:hover:bg-gray-600 {!isCollapsed
```

Comment:
```suggestion
	class="{classNames} group flex h-16 w-6 flex-col items-center justify-center -space-y-1 outline-none *:h-3 *:w-1 *:rounded-full *:hover:bg-gray-300 max-md:hidden dark:*:hover:bg-gray-600 {!isCollapsed
```

`absolute` is already declared in https://github.com/huggingface/chat-ui/blob/navigation-collapse/src/routes/%2Blayout.svelte#L188